README.md

```diff
@@ -1,13 +1,65 @@
-# Media over QUIC Transport
+# Media over QUIC Transport (MoQT)
 
 [![Go Reference](https://pkg.go.dev/badge/github.com/mengelbart/moqtransport.svg)](https://pkg.go.dev/github.com/mengelbart/moqtransport)
 
-`moqtransport` is an implementation of [Media over QUIC
-Transport](https://datatracker.ietf.org/doc/draft-ietf-moq-transport/) on top of
-[quic-go](https://github.com/quic-go/quic-go) and optionally
-[webtransport-go](https://github.com/quic-go/webtransport-go/).
+`moqtransport` is a Go implementation of [Media over QUIC Transport](https://datatracker.ietf.org/doc/draft-ietf-moq-transport/) on top of [quic-go](https://github.com/quic-go/quic-go) and optionally [webtransport-go](https://github.com/quic-go/webtransport-go/).
+
+## Overview
+
+This library implements the Media over QUIC Transport (MoQT) protocol as defined in [draft-ietf-moq-transport-08](https://www.ietf.org/archive/id/draft-ietf-moq-transport-08.txt). MoQT is designed to operate over QUIC and WebTransport for efficient media delivery with a publish/subscribe model.
+
+### Implemented Features
+
+- **Core Protocol Support**: Implements MoQT over both QUIC and WebTransport
+- **Session Management**: Full support for session establishment, initialization, and termination
+- **Control Messages**: Implementation of all control message types defined in the specification
+- **Data Streams**: Support for object delivery via streams and datagrams
+- **Announcement System**: Track announcement and discovery mechanisms
+- **Subscription Handling**: Support for subscribing to tracks and handling subscription updates
+- **Error Handling**: Comprehensive error handling as defined in the specification
+
+### Implementation Status
```

Comment:
This section looks like a draft version of the above. I guess we only need one?

Reply:
Yup